scripts/test.sh

```diff
@@ -8,28 +8,32 @@ Required:
     --website='<WEBSITE-URL>'     URL of the Code Editor instance to test.
 
 Options:
-    -u|--unit-test                Run OSS unit tests.
-    -i|--integ-test               Run OSS integration tests.
-    -s|--style-check              Run OSS style check.
-    -c|--cypress-integ-test       Run Code Editor UI tests.
-    -l|--local                    Run Code Editor UI tests against a local instance (requires -c).
-    -n|--no-patches               Skip automatic patching of OSS.
-    -h|--help                     Show this help message and exit.
+    -w '<WEBSITE-URL>'| --website='<WEBSITE_URL>'       URL of the Code Editor instance to test.
+    -u|--unit-test                                      Run OSS unit tests.
+    -i|--integ-test                                     Run OSS integration tests.
+    -s|--style-check                                    Run OSS style check.
+    -c|--cypress-integ-test                             Run Code Editor UI tests.
+    -l|--local                                          Run Code Editor UI tests against a local instance (requires -c).
+    -n|--no-patches                                     Skip automatic patching of OSS.
+    -h|--help                                           Show this help message and exit.
 """
 }
 
 # Set current project root
 PROJ_ROOT=$(pwd)
 
-# Initialize variables to track if -w, -c, and -l are provided
-W_FLAG_PROVIDED=false
+# Initialize default website value
+WEBSITE="http://localhost:8080"
```

Comment:
you made sure this is the correct port right?

Reply:
I updated it to be `9888` which seems like the default.

The default port never works for me (always says its already in use). I'm not sure if its a me problem or not.